src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -128,11 +135,14 @@ export class ArtworkSidebarBidAction extends React.Component<
     const myLotStanding = artwork.myLotStanding && artwork.myLotStanding[0]
     const hasMyBids = !!(myLotStanding && myLotStanding.most_recent_bid)
 
-    const userNeedsIdentityVerification = bidderNeedsIdentityVerification({
-      sale,
-      user: me,
-      bidder: sale.registrationStatus,
-    })
+    const userLacksIdentityVerification =
+      sale.requireIdentityVerification && !me?.identityVerified
+    const pendingIdentityVerification = me?.pendingIdentityVerification
+
+    const shouldPromptIdVerification =
+      !qualifiedForBidding &&
+      userLacksIdentityVerification &&
+      Boolean(pendingIdentityVerification)
```

Comment:
Maybe the function i have been removing should actually be updated to return an interface full of idv-related properties like this instead.

Reply:
I think this is fine, as long as the login remains confined to this component.